EngineLayer/CrosslinkSearch/BestPeptideScoreNotch.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Proteomics.Fragmentation;
+using Proteomics.ProteolyticDigestion;
+
+namespace EngineLayer.CrosslinkSearch
+{
+    internal class BestPeptideScoreNotch
+    {
+        public BestPeptideScoreNotch(PeptideWithSetModifications bestPeptide, double bestScore, int bestNotch)
```

Comment:
Please remove this class. It is no longer used.

Reply:
done